### PR TITLE
After transaction got deposit/withdrawn transaction in pool section g…

### PR DESCRIPTION
**Pull request by liaojikunwork**
…ot confirmed, but inputfield has not been erased #1297

### CODE REVIEW

**Review by pimato — APPROVED** [review]
tested swap buy/sell, deposit and withdraw and create a market! Worked fine!

**Review by Mi-Lan — APPROVED** [review]
Works good!

File: app/src/components/market/sections/market_buy/market_buy.tsx [hunk 1]
```diff
@@ -77,11 +77,11 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
   const [allowanceFinished, setAllowanceFinished] = useState(false)
   const { allowance, unlock } = useCpkAllowance(signer, collateral.address)
 
-  const hasEnoughAllowance = RemoteData.mapToTernary(allowance, allowance => allowance.gte(amount))
+  const hasEnoughAllowance = RemoteData.mapToTernary(allowance, allowance => allowance.gte(amount || Zero))
```
Comment (KadenZipfel) [inline app/src/components/market/sections/market_buy/market_buy.tsx:80]: Wouldn't this always evaluate to true?

Comment (KadenZipfel) [reply app/src/components/market/sections/market_buy/market_buy.tsx:80]: Just tested. Works fine, no issue

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**